### ISSUE TITLE
Form for submitting poster issues

### ISSUE THREAD

**Issue by darthShadow** [RatingPosterDB/rpdb-folders#43]
Hi,

Can we have a Google Form or similar to submit any poster issues like `tmdb-314367` for example, since it's returning a blank poster with `200` as the response code rather than `404`, which can be handled properly?

Thanks.

**Comment by jaruba** [comment]
@darthShadow 

hi, `tmdb-314367` is an interesting case (btw, please always use the type prefix with TMDB as explained in the API docs, example: `movie-155` or `series-155`, as the type cannot be correctly presumed programmatically for TMDB)

the reason i say that `tmdb-314367` is an interesting case is because this is neither a movie or a series, it was a movie that TMDB incorrectly added in the past: (after which they removed it)
https://www.themoviedb.org/movie/314367-agatha-raisin-and-the-quiche-of-death

which translates to this episode on IMDB:
https://www.imdb.com/title/tt4056574/

if I request this movie from the TMDB API now by ID, I get:
`{"success":false,"status_code":34,"status_message":"The resource you requested could not be found."}`

this is most probably the reason behind the blank poster issues too, but tbh I see no realistic solution to it, RPDB _tries_ to handle any valid IMDB ID and it will even work with IMDB episode IDs (to some extent)

the google form is a good idea, but i think it might be better if i make the report page part of the user web panel at: https://ratingposterdb.com/panel/ (this will be better in terms of spam protection as it requires user authentication)

**Comment by darthShadow** [comment]
> btw, please always use the type prefix with TMDB as explained in the API docs, example: movie-155 or series-155, as the type cannot be correctly presumed programmatically for TMDB IDs

Done. Missed this in the docs and hadn't faced any issues yet, luckily.

> https://ratingposterdb.com/panel/ 

This seems perfect for this, wasn't aware that the panel was even a thing. Maybe I missed the patreon post for this, otherwise I would have suggested this in the first place.

---

Yep, this seems like a difficult scenario. Any suggestions on how to handle this on my side, if it can't be done on the RPDB servers? I guess I could check the TMDB or TVDB API for movies & shows respectively, but that seems excessive. Hoping you have a better suggestion for such scenarios.

---

Off-topic, any reason the panel doesn't have TVDB as an option for Series Poster/Backdrop ?

**Comment by jaruba** [comment]
> Off-topic, any reason the panel doesn't have TVDB as an option for Series Poster/Backdrop ?

yes, it's because the TVDB ID cannot be extracted from the URL pattern

> Yep, this seems like a difficult scenario. Any suggestions on how to handle this on my side, if it can't be done on the RPDB servers? I guess I could check the TMDB or TVDB API for movies & shows respectively, but that seems excessive. Hoping you have a better suggestion for such scenarios.

give me some time to see what can be done about this on the server-side, i have some ideas, this should be an extremely rare case, the issue is actually that this is not a movie at all, it's an episode, the series does have a poster..

the cleanest solution is to disable support for episodes on the server, (as i think i do have data about which of them are episodes) but this is not the first time i see an episode in a user movie folder, so i'll see if this case can be handled better somehow (even if it has already proven itself to be prone to errors)

**Comment by darthShadow** [comment]
>  an episode in a user movie folder

Yeah, I do have plans to eventually go through all movies and remove the ones which are episodes in other shows but that's some time away. In the meantime, I just didn't want the existing poster to be replaced by a blank.

I will leave you to it and wait for a proper solution.
